hikari/api/interaction_server.py

```diff
@@ -107,34 +107,10 @@ class InteractionServer(abc.ABC):
 
     __slots__: typing.Sequence[str] = ()
 
-    @property
-    @abc.abstractmethod
-    def is_alive(self) -> bool:
-        """Whether this interaction server is active.
-
-        Returns
-        -------
-        builtins.bool
-            Whether this interaction server is active
-        """
-
-    @abc.abstractmethod
-    async def close(self) -> None:
-        """Gracefully close the server and any open connections."""
-
-    @abc.abstractmethod
-    async def join(self) -> None:
-        """Wait for the process to halt before continuing."""
```

Comment:
Why tho? This provides a nice interface people can extend upon. If they dont want to inherit it, they can just have it return when called

Reply:
Because how it's run/started/stopped is implementation detail, can't write an interface which is flexible enough to allow for stuff like requiring that they pass an active ASGI server to the "start" method can you and restricting how a sub-component is managed isn't overly helpful when sub-components should only ever be started/run by their parent app/bot